docs/docs/general/data-structures.md

```diff
@@ -32,6 +32,8 @@ data: [{x:'Sales', y:20}, {x:'Revenue', y:10}]
 
 This is also the internal format used for parsed data. In this mode, parsing can be disabled by specifying `parsing: false` at chart options or dataset. If parsing is disabled, data must be sorted and in the formats the associated chart type and scales use internally.
 
+In case you specify the index scale as a number as shown in the first example you will need a labels array. The number indicates the index of the element and does not replace the label as when you provide a string.
```

Comment:
This is only true when the index axis is a category scale.
Maybe something like `The values provided must be parsable by the associated scales or in the internal format of the associated scales. Common mistake would be to provide integers for `category` scale, which uses those as internal format with the meaning of index to the labels array.`

@etimberg, any suggestions?

Reply:
I might tweak the end of that to `which uses integers as an internal format, where each integer represents an index in the labels array`